browser-test/src/admin/admin_program_list.test.ts

```diff
@@ -44,6 +44,24 @@ test.describe('Program list page.', () => {
     await validateScreenshot(page, 'program-list-active-and-draft-versions')
   })
 
+  test('view programs under two tabs - in use and disabled', async ({
+    page,
+    adminPrograms,
+  }) => {
+    await enableFeatureFlag(page, 'disabled_visibility_condition_enabled')
+    await loginAsAdmin(page)
+
+    const publicProgram = 'List test public program'
+    const disabledProgram = 'List test disabled program'
+    await adminPrograms.addProgram(publicProgram)
+    await adminPrograms.addDisabledProgram(disabledProgram)
+
+    await expectProgramListElements(adminPrograms, [publicProgram])
+    await validateScreenshot(page, 'program-list-in-use-tab')
+    await expectDisabledProgramListElements(adminPrograms, [disabledProgram])
```

Comment:
**thought, not-blocking*: This test's behavior would be a little clearer if there was a "navigate to disabled tab" step, but I don't know that refactoring `expectProgramListElements()` to enable that structure is worth the effort.

Reply:
I had the same series of thoughts. It took me a minute to figure out when we were actually navigating to the disabled programs tab, but it wasn't too hard to track down.